src/main/java/seedu/address/model/person/PersonContainsTagPredicate.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.model.person;
+
+import java.util.List;
+import java.util.Set;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+/**
+ * Tests that a {@code Person}'s {@code Tag} matches all the tags given.
+ */
+public class PersonContainsTagPredicate implements Predicate<Person> {
```

Comment:
Perhaps person used in this class could also be changed to classmate to suit Mass Linkers? 🤔